core/opendaq/modulemanager/src/module_manager_impl.cpp

```diff
@@ -779,7 +779,7 @@ ErrCode ModuleManagerImpl::createFunctionBlock(IFunctionBlock** functionBlock, I
                 }
             }
 
-            localIdStr = fmt::format("{}_{}", typeId, maxNum + 1);
+            localIdStr = fmt::format("{}{}", typeId, maxNum + 1);
```

Comment:
This is likely not a good idea. This basically breaks every single setup that contains a function block by changing their local IDs (serialization keys)